cli/src/user.rs

```diff
@@ -161,46 +150,33 @@ impl User {
                     };
                     let mut mls_group = group.mls_group.borrow_mut();
 
-                    let msg = match message {
-                        DsMlsMessage::Ciphertext(ctxt) => {
-                            let unverified_message =
-                                match mls_group.parse_message(ctxt.into(), &self.crypto) {
-                                    Ok(msg) => msg,
-                                    Err(e) => {
-                                        log::error!(
-                                        "Error decrypting MlsCiphertext: {:?} -  Dropping message.",
-                                        e
-                                    );
-                                        continue;
-                                    }
-                                };
-
-                            let processed_message = match mls_group.process_unverified_message(
-                                unverified_message,
-                                None,
-                                &self.crypto,
-                            ) {
-                                Ok(msg) => msg,
-                                Err(e) => {
-                                    log::error!(
-                                        "Error verifying MlsPlaintext: {:?} -  Dropping message.",
-                                        e
-                                    );
-                                    continue;
-                                }
-                            };
-                            processed_message
+                    let unverified_message = match mls_group.parse_message(message, &self.crypto) {
+                        Ok(msg) => msg,
+                        Err(e) => {
+                            log::error!(
+                                "Error decrypting MlsCiphertext: {:?} -  Dropping message.",
```

Comment:
The error message is not quite accurate, in theory it could also be an `MlsPlaintext`. If you want to enforce the wire format (like it was the case until now), you would have to look `message.wire_format()`.

Reply:
Good catch! I'll update the error message.